### DISCUSSION TITLE
上传状态显示失败

### DISCUSSION THREAD

**Discussion by lying813** [Kuingsmile/PicHoro#33] [General]
![IMG_20221119_152924.jpg](https://user-images.githubusercontent.com/40970681/202840059-4ce0669f-78b6-435d-91e3-b8a27629884f.jpg)

**Comment by Kuingsmile** [comment]
@lying813 
请问您使用的图床是哪一个呢，是否可以提供一下报错日志？
日志可以通过`设置-软件日志`查看，右上角图标可导出到剪贴板
